src/windows/FileChooserProxy.js

```diff
@@ -22,12 +22,20 @@ module.exports = {
 
             // file must be copied to local folder to be accessible by app..
             const localFolder = Windows.Storage.ApplicationData.current.localFolder;
-            file.copyAsync(localFolder, file.name, Windows.Storage.NameCollisionOption.replaceExisting)
-            .done(function (savedFile) {
-                successCallback('ms-appdata:///local/' + savedFile.name);
-            }, function(err) {
+
+            // Path to unique folder for uploads within local folder. Encapsulate file selection/upload data from other local data
+            const appUploadFolder = "cordova-filechooser-plugin\\windows\\uploads\\";
```

Comment:
Can we remove the `windows` part of the path? It seems unnecessary.

Reply:
Yes, could do that. My only worry was that it might be a bit ambiguous/confusing. i.e someone might think all uploads go there regardless of platform, whereas it's only a windows thing. Could remove though if you want me to.

Reply:
I think it's fine to remove the windows part. It shouldn't really cause any confusion, unless someone is looking very closely.

Reply:
Okay, will do!